Modules/_cursesmodule.c

```diff
@@ -4140,18 +4159,16 @@ static PyObject *
 _curses_resize_term_impl(PyObject *module, int nlines, int ncols)
 /*[clinic end generated code: output=9e26d8b9ea311ed2 input=2197edd05b049ed4]*/
 {
-    PyObject *result;
-
     PyCursesInitialised;
 
-    result = PyCursesCheckERR(resize_term(nlines, ncols), "resize_term");
-    if (!result)
-        return NULL;
-    if (!update_lines_cols()) {
-        Py_DECREF(result);
-        return NULL;
-    }
+    PyObject *result = PyCursesCheckERR(resize_term(nlines, ncols), "resize_term");
+    CHECK_NOT_NULL_OR_ERROR(result);
```

Comment:
I dislike CHECK_NOT_NULL_OR_ERROR() macro, it makes the code less readable. I prefer the explicit existing `if (!result) return NULL;` code.